code/mapping/ext_modules/cmake_setup.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+import subprocess
+import os
+
+from setup import is_debug_enabled
+
+
+def main():
+    install_instruction = "install"
+    if is_debug_enabled():
+        install_instruction = "develop"
+
+    workingdir = os.path.dirname(os.path.abspath(__file__))
+    subprocess.run(
+        ["python", "setup.py", install_instruction, "--user"], cwd=workingdir
+    )
```

Comment:
The check=True for the subprocess is a valuable addition. Without it the script always exits with a success exit code.

Otherwise I think the error reporting is fine as it is.